selfdrive/modeld/models/dmonitoring.cc

```diff
@@ -13,17 +13,14 @@
 #define MODEL_HEIGHT 640
 #define FULL_W 852 // should get these numbers from camerad
 
-#if defined(QCOM) || defined(QCOM2)
-#define input_lambda(x) (x - 128.f) * 0.0078125f
-#else
-#define input_lambda(x) x // for non SNPE running platforms, assume keras model instead has lambda layer
-#endif
-
 void dmonitoring_init(DMonitoringModelState* s) {
-  const char *model_path = Hardware::PC() ? "../../models/dmonitoring_model.dlc" : "../../models/dmonitoring_model_q.dlc";
-  int runtime = USE_DSP_RUNTIME;
-  s->m = new DefaultRunModel(model_path, &s->output[0], OUTPUT_SIZE, runtime);
   s->is_rhd = Params().getBool("IsRHD");
+
+  const char *model_path = Hardware::PC() ? "../../models/dmonitoring_model.dlc" : "../../models/dmonitoring_model_q.dlc";
```

Comment:
.onnx for pc?
other than this lgtm

Reply:
looks like we can only use .dlc here, the class ONNXModel will rename the suffix ".dlc" to ".onnx".  It might be better to use another PR to refactor ONNXModel, what do you think?

Reply:
yeah, let's clean that up in another PR